### PR TITLE
[DOCS]: Getting Started changes

### PR DESCRIPTION
**Pull request by jmikell821**
### Preview

https://security-docs_114.docs-preview.app.elstc.co/guide/en/security/master/getting-started.html

### CODE REVIEW

File: docs/getting-started/security-ui.asciidoc [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+image::images/overview-pg.png[]
+
+*Histograms*
+
+Time-based histograms show you the number of detections, alerts, and events that have occurred within the selected time range. You can select areas of interest in time-based histograms by selecting a region to reflect a date range, or by selecting a preset value in the timepicker. In the **Stack by** dropdown, you can select specific parameters for which to visualize the individual counts. For example, in the Detection alert trend histogram, stack by `signal.rule.name` to display the total counts by alert name within the specified time frame.
+
+TIP: All Elastic Security histograms, graphs, and tables contain an **Inspect** button so you can examine the {es} queries used to retrieve data throughout
+the app.
+
+Select the collapsable *Timeline* button on the rightmost side of the {security-app} to start an investigation. See <<timeline-ui-overview,>> for more information.
```
Comment (lcawl) [inline docs/getting-started/security-ui.asciidoc:50]:
```suggestion
Select the collapsable *Timeline* button on the rightmost side of the {security-app} to start an investigation. See <<timelines-ui>> for more information.
```

File: docs/getting-started/security-ui.asciidoc [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+[role="screenshot"]
+image::images/events-count.png[]
+
+
+[float]
+[[hosts-ui]]
+=== Hosts page
+
+The Hosts view provides key metrics regarding host-related security events, and a set of data tables that let you interact with the Timeline. See <<hosts-overview,Hosts page overview>> for more information.
```
Comment (lcawl) [inline docs/getting-started/security-ui.asciidoc:64]: This hosts-overview page doesn't seem to exist in the book yet, so I omitted it for now:
```suggestion
The Hosts view provides key metrics regarding host-related security events, and a set of data tables that let you interact with the Timeline.
//See <<hosts-overview,Hosts page overview>> for more information.
```

File: docs/getting-started/security-ui.asciidoc [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+=== Network page
+
+The Network page provides key network activity metrics via an interactive map and network event tables
+that enable interaction with the Timeline. See <<network-page-overview>> for more information. 
+
+[float]
+[[detection-engine-ui]]
+=== Detections page
+
+The Detections page allows you to view and manage all alerts and detection rules to monitor activity within your network. See <<detection-engine-intro>> for more information about detections, prebuilt rules, and alerts.
```
Comment (lcawl) [inline docs/getting-started/security-ui.asciidoc:81]:
```suggestion
The Detections page allows you to view and manage all alerts and detection rules to monitor activity within your network. See <<detection-engine-overview>> for more information about detections, prebuilt rules, and alerts.
```

File: docs/getting-started/security-ui.asciidoc [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+[role="screenshot"]
+image::images/detections-ui.png[]
+
+[float]
+[[timelines-page]]
+=== Timelines page
+
+Use the Timelines page to investigate alerts and complex threats, such as lateral movement of malware across hosts
+in your network. Timelines are responsive and allow you to share your findings among other team members. See <<timeline-ui-overview>> for information about getting started with Timelines.
```
Comment (lcawl) [inline docs/getting-started/security-ui.asciidoc:91]:
```suggestion
in your network. Timelines are responsive and allow you to share your findings among other team members. See <<timelines-ui>> for information about getting started with Timelines.
```

File: docs/getting-started/network-page-overview.asciidoc [hunk 5]
```diff
@@ -0,0 +1,208 @@
+[[network-pg-ov]]
```
Comment (lcawl) [inline docs/getting-started/network-page-overview.asciidoc:1]: Either make this change here or change the links in other pages:
```suggestion
[[network-page-overview]]
```

File: docs/getting-started/index.asciidoc [hunk 6]
```diff
@@ -5,5 +5,6 @@
 
 Looking to get started with Elastic Security? This section describes the Elastic Security UI in Kibana, the system requirements required to run the Elastic Agent with the Elastic Endpoint Security integration, as well as instructions on how to configure and install Elastic Security on your host.
 
-include::siem-ui.asciidoc[]
+include::security-ui.asciidoc[]
+include::network-page-overview.asciidoc[]
```
Comment (lcawl) [inline docs/getting-started/index.asciidoc:9]: This change solved the following error for me:
> ERROR: getting-started/network-page-overview.asciidoc: line 2: invalid part, must have at least one section (e.g., chapter, appendix, etc.)
If that's not the nesting you want, however, let me know and we can play with other options.
```suggestion
include::network-page-overview.asciidoc[leveloffset=+1]
```

File: docs/getting-started/network-page-overview.asciidoc [hunk 7]
```diff
@@ -0,0 +1,208 @@
...
+[chapter]
```
Comment (lcawl) [inline docs/getting-started/network-page-overview.asciidoc:2]: The following error message goes away when you remove the chapter attribute:
> ERROR: getting-started/network-page-overview.asciidoc: line 64: chapter sections do not support nested sections 
```suggestion

```

File: docs/getting-started/security-ui.asciidoc (hunk 2 above)
Comment (lcawl) [reply docs/getting-started/security-ui.asciidoc]: I think this change (omission of the hosts-overview link) is still necessary.

File: docs/getting-started/security-ui.asciidoc [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+[role="screenshot"]
+image::images/hosts-pg.png[]
+
+
+[float]
+[[network-ui]]
+=== Network page
+
+The Network page provides key network activity metrics via an interactive map and network event tables
+that enable interaction with the Timeline. See <<network-page-overview>> for more information.
```
Comment (lcawl) [inline docs/getting-started/security-ui.asciidoc:71]:
```suggestion
that enable interaction with the Timeline. See <<network-pg-ov>> for more information.
```

File: docs/getting-started/security-ui.asciidoc [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+
+The Cases page is used to open and track security issues directly in the {security-app}. For more information, see <<cases-overview>>.
+
+[role="screenshot"]
+image::images/cases-ui-home.png[]
+
+[float]
+[[admin-ui]]
+=== Administration page
+The Administration page allows you to view and manage hosts that are running Elastic Endpoint Security. You can also manage agent integration and monitor the configuration status of your hosts to ensure they are protected. See <<admin-pg-ov>> for more information.
```
Comment (lcawl) [inline docs/getting-started/security-ui.asciidoc:108]: This linked page also fails for me, so I recommend omitting it until management/admin/admin-pg-ov.asciidoc is successfully included in the book:

```suggestion
The Administration page allows you to view and manage hosts that are running Elastic Endpoint Security. You can also manage agent integration and monitor the configuration status of your hosts to ensure they are protected.
// See <<admin-pg-ov>> for more information.
```

File: docs/getting-started/security-ui.asciidoc [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+
+[role="screenshot"]
+image::images/network-ui.png[]
+
+
+[float]
+[[detection-engine-ui]]
+=== Detections page
+
+The Detections page allows you to view and manage all alerts and detection rules to monitor activity within your network. See <<ddetection-engine-overview>> for more information about detections, prebuilt rules, and alerts.
```
Comment (lcawl) [inline docs/getting-started/security-ui.asciidoc:82]: Small typo in link:
```suggestion
The Detections page allows you to view and manage all alerts and detection rules to monitor activity within your network. See <<detection-engine-overview>> for more information about detections, prebuilt rules, and alerts.
```

File: docs/getting-started/security-ui.asciidoc [hunk 11]
```diff
@@ -0,0 +1,110 @@
...
+To save specific filters and queries, click the *Save* button, then **Save current query.**
+
+[role="screenshot"]
+image::images/siem-click-swipe.png[]
+
+[discrete]
+[[page-tabs]]
+== Page tabs
+
+The {security-app} contains seven page tabs that enable analysts to view, analyze, and manage security data:
```
Comment (benskelker) [inline docs/getting-started/security-ui.asciidoc:22]:
```suggestion
The {security-app} contains seven page tabs that enable analysts to view, analyze, and manage security data:

```
Add a new line so the bullets render correctly.

File: docs/getting-started/network-page-overview.asciidoc [hunk 12]
```diff
@@ -0,0 +1,207 @@
...
+There are also tabs for viewing and investigating specific types of data:
+
+* Flows: source and destination IP addresses and countries
+* DNS: DNS network queries
+* HTTP: received HTTP requests (HTTP requests for applications using
+{apm-app-ref}/apm-getting-started.html[Elastic APM] are monitored by default)
+* TLS: handshake details
+* Anomalies: anomalies discovered by <<machine-learning, machine learning jobs>>
+* <<det-engine-terminology, External alerts>>: alerts received from
+external monitoring tools, such as Elastic Endpoint Security
```
Comment (benskelker) [inline docs/getting-started/network-page-overview.asciidoc:26]:
```suggestion
external monitoring tools, such as Suricata
```

File: docs/getting-started/network-page-overview.asciidoc [hunk 13]
```diff
@@ -0,0 +1,207 @@
+[[network-page-overview]]
+= Network page overview
+
+The Network view provides key network activity metrics and an interactive map, facilitates investigation time enrichment, and provides network event tables that enable interaction with the Timeline. You can drag and drop items of interest from the Network view to Timeline for further investigation.
```
Comment (benskelker) [inline docs/getting-started/network-page-overview.asciidoc:4]: `facilitates investigation time enrichment`

@jmikell821 
I know that's my fault (it's awful, and I've no idea what it means 😬), but please remove or change it - or remind me to when you're done. Thanks

Comment (jmikell821) [reply docs/getting-started/network-page-overview.asciidoc:4]: @benskelker I was kinda wondering what this meant - LOL! Will think of another sentence.

**Pull request merged**